utils.py

```diff
@@ -77,3 +79,159 @@ def decorated_function(*args, **kwargs):
         return decorated_function
 
     return decorator
+
+
+'''
+    Requirements for auth:
+        SwagLyrics PEM should be saved as "swag_pem" in environment variables
+        APP_ID is the id of the github app
+        repo_url is the api url of the repo this app is installed on
+'''
+
+APP_ID = os.getenv('swag_appid')
+
+# WARN: Security vulnerabilities - auth methods should eventually be moved
```

Comment:
elaborate?

Reply:
I'm not 100% sure if this would be considered a security vulnerability but to ensure that there is only ever one authorized JWT or installation access token, we have to store the tokens somewhere outside of a method. We're currently storing them as global variables, which can lead to unexpected side effects.